stix2/__init__.py

```diff
@@ -54,7 +54,7 @@
     WithinQualifier,
 )
 from .utils import new_version, revoke
-from .v20 import *  # This import will always be the latest STIX 2.X version
+from .v21 import *  # This import will always be the latest STIX 2.X version
```

Comment:
This is purposely meant to be `.v20` for now until we make a major release of the library (once STIX 2.1 becomes a CS). The idea is that we are providing the objects to aid the debugging and finding problems on the spec, but the 2.1 is not stable yet. Text from the README follows:

STIX 2.X Technical Specification Support
This version of python-stix2 brings initial support to STIX 2.1 currently at the CSD level. The intention is to help debug components of the library and also check for problems that should be fixed in the specification.

The stix2 Python library is built to support multiple versions of the STIX Technical Specification. With every major release of stix2 the import stix2 statement will automatically load the SDO/SROs equivalent to the most recent supported 2.X Committee Specification. Please see the library documentation for more details.

```suggestion
from .v20 import *  # This import will always be the latest STIX 2.X version
```

Reply:
I thought the point of the stix2.1 branch was to prepare for that major release.  So everything on that branch should be as if we are preparing for that release.  That's how we test it and make sure it will work.  From that standpoint, the README is wrong and needs to be updated at some point (though that's probably low priority for now).  If people want a version where 2.0 is the default, they should use the master branch.

I had to make this change because the workbench was pulling in all the wrong versions of the stix classes.  Well, wrong on the 2.1 branch because I thought it should default to 2.1 classes, and the v21 workbench test suite definitely needed 2.1 classes.  It seemed clear to me that the intent was to default to 2.1, so it was the v20 workbench test suite that needed to be removed.  If not, we need to reinstate the v20 workbench test suite, and remove the v21 test suite.  But that seems like it defeats the purpose of having a stix2.1 branch.

So... I don't understand why you want to do this.

Reply:
The plan is that when we merge this, we actually release something like `v1.2.0` which matches the latest (to be approved) STIX 2.1 CSD 02. When STIX 2.1 is approved as a CS we would then change the import statement to load 2.1 objects automatically `v2.0.0` along with any other changes to align our library to the spec.

I am ok with reinstating the v20 tests even if that means we don't test the workbench against v21 for now. Though, I don't know how much or what other areas will be affected by this change.

Reply:
> when we merge this

This means merging stix2.1 to master?  Ok, hadn't realized we were going to make another official minor version release, with the stix2.1 branch merged to master, before STIX 2.1 was finalized.  I thought this branch was to be the next major version of the library, and the first official release with any STIX 2.1 support at all.

> Though, I don't know how much or what other areas will be affected by this change.

Switching test suites wouldn't change anything core to the codebase (as long as I can make it work with the same trick I used with the v21 test suite).  Changing the import probably wouldn't cause widespread breakage either; I didn't run into any issues with the old import until I started looking into the workbench test suite.  I'll give it a shot.